core/sawtooth/cli/admin_sub/clean.py

```diff
@@ -1,157 +0,0 @@
-# Copyright 2016 Intel Corporation
```

Comment:
We should leave clean's implementation until it's moved/implemented for 0.8 CLI.